sdk-base/src/main/java/com/mapbox/maps/dsl/CameraOptionsKtx.kt

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.maps.dsl
+
+import com.mapbox.maps.CameraOptions
+
+/**
+ * DSL builder function to create [CameraOptions] object.
+ */
+inline fun cameraOptions(block: CameraOptions.Builder.() -> Unit) =
```

Comment:
The current implementation has limitations that we didn't make use of DslMarker, which makes it not safe when nested with other DSL builders. We can bypass this issue by extending the BuilderClass like: 

```
@DslMarker
annotation class CameraOptionsDsl

@CameraOptionsDsl
class CameraOptionsBuilderDsl : CameraOptions.Builder

inline fun cameraOptions(buildCompany: CameraOptionsBuilderDsl.() -> Unit): CameraOptions {
```

But since the CameraOptions Builder is a final class and can not be extended. This wouldn't work.

cc @kiryldz @tobrun